comms/EclCC.cpp

```diff
@@ -725,11 +725,26 @@ boost::filesystem::path EclCCSubPath(const boost::filesystem::path & directory)
     return directory / "clienttools" / "bin" / "eclcc.exe";
 }
 
-unsigned int FindAllCEclCC(const boost::filesystem::path & progFiles, CEclCCVector & results)
+void PushUniqueEclCC(CEclCCVector& results, boost::filesystem::path eclccPath)
```

Comment:
Should be `const boost::filesystem::path &eclccPath`

Reply:
Done.